carousel/src/carousel/edit.js

```
@@ -0,0 +1,355 @@
+			}
+
+			let count = 0;
+
+			switch ( contentBlock.name ) {
+				case 'core/gallery':
+				case 'core/group':
+					count = contentBlock.innerBlocks?.length || 0;
+					break;
+				case 'core/query':
+				case 'woocommerce/product-collection':
+					const { getTaxonomy } = select( 'core' );
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)